templates/grafana_agent.yaml

```
@@ -0,0 +1,110 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana-agent
+  namespace: kube-system
+  labels:
+    juju.io/application: kube-ovn
```

this probably needs to be templated and rendered by the charm.  the application name can be anything

```suggestion
    juju.io/application: {juju_app}
```